pom.xml

```diff
@@ -2238,6 +2239,44 @@
       </activation>
     </profile>
 
+    <profile>
+      <id>spark3.0</id>
+      <properties>
+        <spark3.version>${spark30.version}</spark3.version>
+        <spark.version>${spark3.version}</spark.version>
+        <sparkbundle.version>3.0</sparkbundle.version>
+        <scala.version>${scala12.version}</scala.version>
+        <scala.binary.version>2.12</scala.binary.version>
+        <hudi.spark.module>hudi-spark3.0.x</hudi.spark.module>
+        <hudi.spark.common.modules.1>hudi-spark3-common</hudi.spark.common.modules.1>
+        <kafka.version>${kafka.spark3.version}</kafka.version>
```

Comment:
Do we need `<scalatest.version>${scalatest.spark3.version}</scalatest.version>`?

Reply:
Seems it's not added.  But I assume it's not really necessary here.